web/utils.js

```diff
@@ -439,3 +439,173 @@ function checkEnter(event) {
         $("#sbox").submit();
     }
 }
+
+// Intelligence Window code starts from here
+document.onmousemove = function(event) {
+    if (typeof event !== 'undefined') {
+        document.intelliWindowMouseX = event.clientX;
+        document.intelliWindowMouseY = event.clientY;
+    }
+};
+
+$(document).keypress(function(e) {
+    if (document.activeElement.id === 'search' ||
+        typeof document.intelliWindow === 'undefined') {
+        return true;
+    }
+
+    if (e.which === 49) { // '1' pressed
+        if (document.intelliWindow.className === "intelli_window_style") {
+            hideIntelliWindow();
+        } else if (document.intelliWindow.className === "intelli_window_style_hide") {
+            showIntelliWindow();
+        }
+    }
+    if (e.which === 50) { // '2' pressed
+        var symbol = document.intelliWindow.symbol;
+        var highlighted_symbols_with_same_name = $("a").filter(function(index) {
+            return $(this).text() === symbol &&
+                $(this).css("background-color") === "gold";
+        })
+        if (highlighted_symbols_with_same_name.length === 0) {
+            highlightSymbol(symbol);
+        } else {
+            unhighlightSymbol(symbol);
+        }
+    }
+    return true;
+});
+
+function onMouseOverSymbol(symbol, symbolType) {
+    updateIntelliWindow(symbol, symbolType);
```

Comment:
Symbol highlighting feature is fantastic!
But I'm afraid nobody will ever discover this feature is hidden under 1,2 key press.

What about automatic highlighting symbol under cursor:
function onMouseOverSymbol(symbol, symbolType) {
    updateIntelliWindow(symbol, symbolType);
    
    //highlight symbol under cursor
    unhighlightAll();
    highlightSymbol(symbol);    
}

I tested this change and it works really nice.
It's not much annoying for user that don't care about this and it's very helpful for anyone trying to read the code in browser.